### DISCUSSION TITLE
Flutils(Flet-Utils)

### DISCUSSION THREAD

**Discussion by ndonkoHenri** [flet-dev/flet#496] [Show and tell]
I built a tool for Flet's community.

Check it out..and eventually share your impressions with me.

**Repo**: https://github.com/ndonkoHenri/Flet-Samples/tree/master/Flet-Utils

**Online version**(link hopefully not broken 🙂): https://flutils.fly.dev/

**Sample**:
https://user-images.githubusercontent.com/98978078/196046824-1b91150c-acb8-4493-8b29-b72162f324ae.MP4

**Comment by FeodorFitsner** [comment]
Would it make sense to break [utils.py](https://github.com/ndonkoHenri/Flet-Samples/blob/master/Flet-Utils/utils.py) into multiple files? ;)

**Reply by ndonkoHenri** [reply]
I think that's a good idea. It's close to 2000 code lines already.

Thanks.

(Just to ask: Does Single responsibility principle also apply on files ?)
